.wp-env.json

```diff
@@ -4,6 +4,9 @@
 	"themes": [ "./test/emptytheme" ],
 	"env": {
 		"tests": {
+			"config": {
+				"SCRIPT_DEBUG": true
+			},
```

Comment:
I'm not entirely sure about this. I think it's good for the dev environment to have this enabled by I believe that ideally the tests environment should be as close to prod as possible.

Also, for performance tests, this will have an impact.

Reply:
This is temporary to see if anything fails in the development environment when StictMode is enabled.